src/components/common/form/message-item/MessageItem.spec.jsx

```diff
@@ -0,0 +1,49 @@
+import { screen } from '@testing-library/react';
+import { renderWithTimecardContext } from '../../../../test/helpers/TimecardContext';
+
+import MessageItem from './MessageItem';
+
+describe('MessageItem', () => {
+  window.HTMLElement.prototype.scrollIntoView = jest.fn();
+  it('should render a summary component with all messages details', () => {
```

Comment:
Should there also be a test for doubleDateMoved?

Reply:
Test added for double date scenario